orte/mca/qos/base/qos_base_frame.c

```diff
@@ -0,0 +1,118 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
+/*
+ * Copyright (c) 2014      Intel, Inc. All rights reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#include "orte_config.h"
+#include "orte/constants.h"
+
+#include "opal/class/opal_bitmap.h"
+#include "opal/mca/mca.h"
+#include "opal/util/output.h"
+#include "opal/mca/base/base.h"
+
+#include "orte/mca/rml/base/base.h"
+#include "orte/mca/qos/base/base.h"
+#include "orte/mca/qos/qos.h"
+#if OPAL_ENABLE_FT_CR == 1
+#include "orte/mca/state/state.h"
+#endif
+
+/*
+ * The following file was created by configure.  It contains extern
+ * statements and the definition of an array of pointers to each
+ * component's public mca_base_component_t struct.
+ */
+
+#include "orte/mca/qos/base/static-components.h"
+
+/*
+ * Global variables
+ */
+orte_qos_base_t orte_qos_base;
+OPAL_TIMING_DECLARE(tm_qos)
+
+static int orte_qos_base_register(mca_base_register_flag_t flags)
+{
+#if OPAL_ENABLE_TIMING
+    /* Detailed timing setup */
+    orte_qos_base.timing = false;
+    (void) mca_base_var_rtegister ("orte", "qos", "base", "timing",
```

Comment:
Please correct misprint: mca_base_var_rtegister => mca_base_var_register. It doesn't compilable with --enable-timing.